api/lib/validation/schemaValidator.js

```diff
@@ -57,7 +57,7 @@ var getDaysInMonthInISOFormat = function() {
 };
 
 var getMetricTypes = function() {
-  var metricTypeEnum = ['memoryused'];
+  var metricTypeEnum = ['memoryutil', 'responsetime', 'throughput'];
```

Comment:
Also ApiServer will need to know about the collection method for the upper limit and metric type validations.